pkg/service/credential/model.go

```diff
@@ -24,6 +24,7 @@ type CreateCredentialRequest struct {
 	Expiry      string         `json:"expiry,omitempty"`
 	Revocable   bool           `json:"revocable,omitempty"`
 	Suspendable bool           `json:"suspendable,omitempty"`
+	Evidence    []any          `json:"evidence,omitempty"`
```

Comment:
should we create another type here that always has `id` and `type` fields?

Reply:
Yea thats a good way to do it, i'll make the change

Reply:
I would suggest we go even further, and only allow specific types of evidence. That way, we can validate that the input and shape is correct.